test/runtests.jl

```diff
@@ -366,6 +366,10 @@ mkpath(TEST_EXAMPLES_DIR)
     include("rectilinear.jl")
   end
 
+  @testset "pvtk files" begin
```

Comment:
```suggestion
  @testset "Parallel VTK (PVTK) files" begin
```
Just a little more information for users who are not as familiar with the parallel file formats.